tutorials/tutorial-01.qmd

```diff
@@ -6,33 +6,1403 @@ time: "~45 minutes"
+```
+
+### Preparing Geometries for Visualisation
+
+To visualise the results, we need to combine the intersection flags with the MPA geometries. We also simplify the geometries because marine protected area boundaries often contain highly detailed coastlines with thousands of vertices, which slows down plotting.
+
+We temporarily disable [`s2` (spherical geometry)](../concepts.qmd#the-geospatial-software-stack) because `st_simplify()` uses the GEOS library, which expects planar coordinates. Since our data is already in a projected CRS (EPSG:3035), this is safe:
+
+```{r}
+#| label: prepare-visualisation-data
+# Add intersection flags to MPA geometries and simplify
+sf_use_s2(FALSE)
+
+mpa_simple <- mpa_combined |>
+  bind_cols(intersections) |>
+  st_simplify(dTolerance = 2000, preserveTopology = TRUE)
+
+sf_use_s2(TRUE)
+```
+
+The `dTolerance` parameter specifies the simplification tolerance in CRS units. Since we're using EPSG:3035 (metric), `dTolerance = 2000` means 2 km, imperceptible at our map scale but dramatically faster to render.
+
+```{r}
+#| label: compare-simplification
+# Compare object sizes as proxy for complexity reduction
+original_size <- object.size(mpa_combined)
+simplified_size <- object.size(mpa_simple)
+
+sprintf(
+  "Object size reduced from %s to %s (%.0f%% reduction)",
+  format(original_size, units = "MB"),
+  format(simplified_size, units = "MB"),
+  100 * (1 - as.numeric(simplified_size) / as.numeric(original_size))
+)
+```
+
+::: {.callout-note}
+Always perform spatial analysis on the original geometries, then simplify for visualisation. Simplifying before analysis can introduce errors in intersection tests.
+:::
+
+### Visualising the Results
+
+#### Map of All Protected Areas
+
+First, let's create a base map showing all protected areas, coloured by whether they intersect with infrastructure. Compare with @fig-protected-areas-preview which showed the three MPA types before analysis:
+
+```{r}
+#| label: fig-mpas-affected
+#| fig-cap: "Protected areas in the North Sea coloured by infrastructure intersection status"
+#| fig-height: 7
+
+ggplot() +
+  geom_sf(data = europe, fill = "gray95", colour = "gray70") +
+  geom_sf(
+    data = mpa_simple,
+    aes(fill = any, colour = any),
+    linewidth = 0.1,
+    alpha = 0.3
+  ) +
+  scale_fill_manual(
+    values = c("FALSE" = "lightgreen", "TRUE" = "coral"),
+    labels = c("No intersection", "Intersects infrastructure"),
+    name = NULL
+  ) +
+  scale_colour_manual(
+    values = c("FALSE" = "lightgreen", "TRUE" = "coral"),
+    guide = "none"
+  ) +
+  coord_sf(
+    xlim = c(bbox_ext["xmin"], bbox_ext["xmax"]),
+    ylim = c(bbox_ext["ymin"], bbox_ext["ymax"])
+  ) +
+  labs(
+    title = "Protected Areas and Infrastructure Overlap",
+    subtitle = "North Sea region"
+  ) +
+  theme_minimal() +
+  theme(legend.position = "bottom")
+```
+
+#### Map with Infrastructure Overlay
+
+Now let's add the infrastructure layers to see the spatial pattern:
```

Comment:
could we put the infrastructure _below_ the MPA?

Reply:
I prefer this way to be honest